---
title: "Download"
subtitle: "Workshop on Data Visualization in R"
author: "`r paste0('<b>NBIS</b> â€¢ ',format(Sys.time(), '%d-%b-%Y'))`"
---

```{r,child="assets/header-lab.Rmd"}
```

```{r,include=FALSE}
# data handling
library(dplyr)
#library(tidyr)
#library(stringr)

# plotting
library(ggplot2)

#library(biomaRt) # annotation
#library(DESeq2) # rna-seq
#library(edgeR) # rna-seq
```


The following data will be used in the used for the exercises in this course. The data comes from a mock RNA sequencing data with 12 samples that are from cell-lines. As in any normal RNAseq analysis the counts of genes were normalized using different methods such as `VST` and `CPM` after filtering out the genes that were of very low expression. We would use these data for visualizations.

**Download all of these files into your current working directory**

You can download all the files together [here](data/data.zip).
Make a directory called `data` and `unzip` in that directory!

If `workshop_on_plotting_in_R` is the current working directory, the directory tree should look like this:

* <span style="color:blue">ggplot_geneco_course</span>
  + <span style="color:blue">data</span>
    + counts_deseq2.txt
    + counts_filtered.txt
    + counts_raw.txt
    + counts_vst.txt
    + metadata.csv
    + human_biomaRt_annotation.csv
    + Time_t24_vs_t0.txt
    + Time_t2_vs_t0.txt
    + Time_t6_vs_t0.txt
    + shiny_app_data.csv
    + Blood_Cells_Image.jpeg

You can find the information related to the important files below:

# Counts tables

* Table with raw gene counts: [Raw Counts](data/counts_raw.txt)
* Table with gene counts after filtering: [Filtered Counts](data/counts_filtered.txt)
* Table with gene counts normalized with VST: [VST counts](data/counts_vst.txt)
* Table with gene counts normalized with DESeq2: [DESeq2 counts](data/counts_deseq2.txt)

# Metadata

* Metadata of the samples: [Sample Metadata](data/metadata.csv)
* Metadata of the genes with their functions: [Gene Annotation](data/human_biomaRt_annotation.csv)

# DE genes

Below are the lists of differentially expressed genes between different time points

* [Time point 2 vs Time point 0](data/Time_t2_vs_t0.txt)
* [Time point 6 vs Time point 0](data/Time_t6_vs_t0.txt)
* [Time point 24 vs Time point 0](data/Time_t24_vs_t0.txt)

# Shiny App data

For the sake of making a shiny app the real time data of covid related scenarios for handful of countries have been in a specific period starting from the onset of the pandemic until few months after the start of the global vaccination campaign has been downloaded from [Ourworldindata](https://ourworldindata.org/).

* covid data table for the ShinyApp [here](data/shiny_app_data.csv)

***
